src/components/index.d.ts

```diff
@@ -476,21 +476,21 @@ export declare const ZipInput: {
   (props: any): JSX.Element
   PUBLIC_PROPS: {
     'data-tid': any
-    disabled: any
-    allCaps: any
-    name: any
     labelCopy: any
-    validator: any
-    initialValue: any
+    name: any
```

Comment:
It looks like the propTypes for `data-tid`, `labelCopy`, and `name` are `PropTypes.string.isRequired` - is there a reason why we'd use `any` here?

Reply:
silly mistake -- fixed in 9d92c639b2d0d495b587646007e327ebcbb78876 thanks for catching